examples/basic/run.sh

```diff
@@ -11,6 +11,6 @@ docker run \
   -v $DOCKER_SOCKET:$DOCKER_SOCKET \
   -v $OREMDA_DIR:/oremda \
   -v $RUNNER_DIR:/runner \
-  -v $PLASMA_DIR:$PLASMA_DIR \
+  -v $PLASMA_DIR:/tmp \
```

Comment:
Not really, assumptions are made in parts of the code that the `DEFAULT_OREMDA_VAR_DIR=/tmp` is to be used.
If we don't mount it explicitly to `/tmp` things would break as soon as you change the host mount to anything other than `/tmp`